doc/source/getting-started/add-license-headers/issues-limitations.rst

```diff
@@ -0,0 +1,15 @@
+Known issues and limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. dropdown:: After running the hook, I am seeing a "Skipped unrecognised file" message
```

Comment:
We are supposed to use American (versus British) spellings. The message should spell as unrecognized.